__tests__/fixtures/project-sample-new.json

```diff
@@ -0,0 +1,318 @@
+              "node": {
+                "type": "element",
+                "content": {
+                  "elementType": "container",
+                  "children": [
+                    {
+                      "type": "static",
+                      "content": "Page 3"
+                    }
+                  ]
+                }
+              },
+              "value": "contact-us",
+              "reference": {
+                "type": "dynamic",
+                "content": {
+                  "referenceType": "state",
+                  "id": "route"
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "components": {
+    "OneComponent": {
+      "name": "OneComponent",
+      "propDefinitions": {
+        "title": {
+          "type": "string",
+          "defaultValue": "Hello"
+        },
+        "items": {
+          "type": "array",
+          "defaultValue": []
+        }
+      },
+      "stateDefinitions": {
+        "isVisible": {
+          "type": "boolean",
+          "defaultValue": true
+        }
+      },
+      "node": {
+        "type": "element",
+        "content": {
+          "elementType": "container",
+          "attrs": {
+            "data-static-attr": {
+              "type": "static",
+              "content": "test"
+            },
+            "data-dynamic-attr": {
+              "type": "dynamic",
+              "content": {
+                "referenceType": "prop",
+                "id": "title"
+              }
+            }
+          },
+          "children": [
+            {
+              "type": "element",
+              "content": {
+                "elementType": "text",
+                "children": [
+                  {
+                    "type": "static",
+                    "content": "Hello World!"
+                  },
+                  {
+                    "type": "dynamic",
+                    "content": {
+                      "referenceType": "prop",
+                      "id": "title"
+                    }
+                  }
+                ]
+              }
+            },
+            {
+              "type": "repeat",
+              "content": {
+                "node": {
+                  "type": "element",
+                  "content": {
+                    "elementType": "text",
+                    "children": [
+                      {
+                        "type": "dynamic",
+                        "content": {
+                          "referenceType": "local",
+                          "id": "item"
+                        }
+                      },
+                      {
+                        "type": "element",
+                        "content": {
+                          "elementType": "list",
+                          "attrs": {
+                            "items": {
+                              "type": "static",
+                              "content": ["angular", "react", "vue"]
```

Comment:
Made this: https://github.com/teleporthq/teleport-code-generators/issues/136 to discuss the status of arrays passed down as attributes/props.